docs/spec-v1alpha1.md

```diff
@@ -0,0 +1,221 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [OCI Model Configuration Specification](#oci-model-configuration-specification)
+  - [Terminology](#terminology)
+    - [Tensor](#tensor)
+    - [GitRepo](#gitrepo)
+  - [Properties](#properties)
+  - [Example](#example)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# OCI Model Configuration Specification
+
+An OCI Model is a Machine Learning/Deep Learning model. This specification outlines the JSON format describing models for use and execution tool and its relationship to filesystem changesets.
+
+This section defines the `application/vnd.caicloud.model.config.v1alpha1+json` media type.
+
+## Terminology
+
+This specification uses the following terms:
+
+### [Tensor](https://www.tensorflow.org/guide/tensor)
+
+Tensors are multi-dimensional arrays with a uniform type.
+
+- **name** string, OPTIONAL
+
+    Name of the tensor.
+
+- **description** string, OPTIONAL
+
+    Description of the tensor.
+
+- **dtype** string, OPTIONAL
+
+    DType of the tensor.
+
+- **size** object, OPTIONAL
+
+    Size of the tensor.
+
+### [GitRepo](https://kubernetes.io/docs/concepts/storage/volumes/#gitrepo)
+
+GitRepo is the Git repository.
+
+- **repository** string, OPTIONAL
+
+    Repository of the training scripts
+
+- **revision** string, OPTIONAL
+
+    Revision of the repository
+
+- **description** string, OPTIONAL
+
+    Description of the training script
+
+## Properties
+
+Note: Any OPTIONAL field MAY also be set to null, which is equivalent to being absent.
+
+- **created** string, OPTIONAL
```

Comment:
is this a well-established field in other specs?  It's not intuitive to know what `created` mean by just looking at the field name..

Reply:
It is a well-established field in OCI image spec

Reply:
ok